api/serverV2.go

```diff
@@ -158,3 +158,8 @@ func (svr *ServerV2) Stop(ctx context.Context) error {
 func (svr *ServerV2) ReceiveBlock(blk *block.Block) error {
 	return svr.core.ReceiveBlock(blk)
 }
+
+// CoreService returns the coreservice of the api
+func (svr *ServerV2) CoreService() CoreService {
+	return svr.core
```

Comment:
i don't see you delete `APIServer` in chainservice.go?

Reply:
this is done by  https://github.com/iotexproject/iotex-core/pull/3259